devel/management/commands/retire_user.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+retire_user
+
+Retire an user by de-activing the account and moving the user to the retired
+group.
+
+Usage ./manage.py retire_user user
+"""
+
+import logging
+import sys
+
+from django.contrib.auth.models import User, Group
+from django.core.management.base import BaseCommand, CommandError
+
+from devel.models import UserProfile
+
+
+logging.basicConfig(
+    level=logging.WARNING,
+    format=u'%(asctime)s -> %(levelname)s: %(message)s',
+    datefmt=u'%Y-%m-%d %H:%M:%S',
+    stream=sys.stderr)
+logger = logging.getLogger()
+
+MAPPING = {
+        'Developers': 'Retired Developers',
+        'Trusted Users': 'Retired Trusted Users',
+        'Support Staff': 'Retired Support Staff',
+}
+
+
+class Command(BaseCommand):
+    help = "Retires a user, deactiving the user and moving the group membership to retired groups."
```

Comment:
May replace the comma with a "by" so that it's more clear that the second part is performed by this function?

Reply:
Fixed